packages/next/src/build/index.ts

```diff
@@ -2813,7 +2828,7 @@ export default async function build(
                   SERVER_DIRECTORY,
                   'app'
                 )
-                if (fsExistsSync(originalServerApp)) {
+                if (await fileExists(originalServerApp)) {
```

Comment:
I think this might not be faster than fs.`existsSync` as it needs to access the file and handle the error. x-ref: https://github.com/vercel/next.js/pull/55675

Reply:
That's true, but the Node.js documentation does recommend using the `fs.stat` instead (used internally by our `fileExists` helper).